### ISSUE TITLE
`Key` parse should query `Definition` first

### ISSUE THREAD

**Issue by loggerhead** [loggerhead/Easy-Karabiner#1]
For example

``` python
['alt mouse_left', 'mouse_left Open::COPY_FINDER_PATH']
```

Not works as expect, but generated XML config below

``` XML
<autogen> __KeyToKey__
  PointingButton::LEFT, ModifierFlag::OPTION_L, ModifierFlag::NONE,
  PointingButton::LEFT, Open::COPY_FINDER_PATH
</autogen>
```

**Comment by loggerhead** [comment]
This bug is caused by a design problem and fix it will rewrite a lot of current codes, so it will NOT fix in 0.4.x version, but it will fix in 0.5.x version(That is, Easy-Karabiner2); and 0.5.x will not compatible with 0.4.x for this reason.

**Comment by loggerhead** [comment]
Fixed in 0.5.x
